webservices/legal_docs/archived_murs.py

```diff
@@ -23,15 +23,17 @@
 ALL_ARCHIVED_MURS = """
     SELECT DISTINCT
         mur_id,
-        mur_number as mur_no
+        mur_number as mur_no,
+        mur_id as case_serial
     FROM MUR_ARCH.ARCHIVED_MURS
-    ORDER BY mur_id
+    ORDER BY mur_id desc
```

Comment:
Just like cases and ao,  should you also order by case_serial for archived murs?

Reply:
Thanks for review.
It is better sort by column itself not alias name.
case_serial is the alias name of mur_id.